content/docs/concepts/security/index.md

```diff
@@ -10,98 +10,160 @@ aliases:
-as the service account 'frontend-team' and service 'backend' running as the service account 'backend-team'. Istio supports services running
-on both Kubernetes containers and VM/bare-metal machines.
+This guide provides a high-level overview of Istio security; you can find more information about specific Istio security concepts in the other guides in this section. To find out how to get started using Istio security with deployed services, see the Istio Security Basics tutorial. For more detailed howtos on security tasks, see our Security Tasks.
+
+## High-Level Architecture and Features
+
+Istio security involves multiple components, including Citadel for key and certificate management,
+Envoy and perimeter proxies for implementing secure communication between clients and servers,
+Pilot for distributing authentication policies and secure naming information (distributed with routing information)
+to the proxies, and Mixer for managing auditing.
 
 {{< image width="80%" ratio="56.25%"
-    link="./auth.svg"
+    link="./architecture.svg"
     alt="Components making up the Istio security model."
     caption="Istio Security Architecture"
     >}}
 
-As illustrated in the diagram, Istio leverages secret volume mount to deliver keys/certs from Citadel to Kubernetes containers. For services running on
-VM/bare-metal machines, we introduce a node agent, which is a process running on each VM/bare-metal machine. It generates the private key and CSR (certificate
-signing request) locally, sends CSR to Citadel for signing, and delivers the generated certificate together with the private key to Envoy.
+### PKI (Public Key Infrastructure)
 
-## Mutual TLS authentication
+Istio PKI, built on top of Istio Citadel, is responsible for securely provisioning a strong workload identity in the form of X.509 certificate in SPIFFE format to every workload, and automating key & certificate rotation and revocation at scale. 
 
-### Identity
+### Authentication
 
-Istio uses [Kubernetes service accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) to identify who runs the service:
+Istio provides two different types of authentication:
 
-*   A service account in Istio has the format "spiffe://\<_domain_\>/ns/\<_namespace_>/sa/\<_serviceaccount_\>".
+* **Service-to-service Authentication** enables data-in-transit encryption and strong channel-level authentication using mutual TLS.
 
-    * _domain_ is currently _cluster.local_. We will support customization of domain in the near future.
-    * _namespace_ is the namespace of the Kubernetes service account.
-    * _serviceaccount_ is the Kubernetes service account name.
+* **User Authentication** enables request-level authentication with JSON Web Token (JWT) validation
+  and a streamlined developer experience for Auth0, Firebase Auth, Google Auth, and custom Auth.
```

Comment:
One of these words needs to be added to the top level .spelling file to get the gat moving

Reply:
Done.